src/main/java/Parser.java

```diff
@@ -0,0 +1,80 @@
+import java.util.Scanner;
+import java.time.format.DateTimeParseException;
+
+public class Parser {
+    Scanner sc = new Scanner(System.in);
```

Comment:
Doesn't the scanner exist in another class as well? How about thinking of reducing the number of scanners?